pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -11223,9 +11223,9 @@
       a:
         body_01: "The Hour of Code activities page includes a [filter for activities compatible with screen readers](%{filter_url})."
         body_02: "If you use a screen reader, the [Quorum tutorial for beginners](%{beginners_url}) or the [Quorum tutorial for comfortable students](%{comfortable_url}) are great places to start an Hour of Code. Quorum started as an interpreted language originally designed to be accessible through screen readers. It has become a general purpose programming language designed for any user. And, if you want to go beyond an Hour of Code, the team at Quorum has additional tools and curriculum."
-        body_03: "If you use a screen reader and are interested in robotics, try the [Dash Joins a Dance Circle activity](%{k5_url}) (for grades K-5) or the [Dash Joins a Dance Circle with Functions activity](%{6_8_url}) (for grades 6-8) from Lauren Milne."
+        body_03: "If you use a screen reader and are interested in robotics, try the [Dash Joins a Dance Circle activity](%{k5_url}) (for grades K-5) or the [Dash Joins a Dance Circle with Functions activity](%{six_8_url}) (for grades 6-8) from Lauren Milne."
```

Comment:
Because we're not changing the number of interpolations, this might be okay, but we'll need to test it.

Reply:
This one results in broken links though

Reply:
They were broken in Spanish when I merged in the newest down/out

Reply:
Just checked Spanish and they are working again with the new strings.